ipinfo/handler_utils.py

```diff
@@ -50,15 +52,14 @@ def get_headers(access_token):
     return headers
 
 
-def format_details(details, countries):
+def format_details(details, countries,eu_countries):
     """
     Format details given a countries object.
 
     The countries object can be retrieved from read_country_names.
     """
-    json_details = countries.get(details.get("country"))
-    details["country_name"] = json_details["name"] if json_details else None
-    details["isEU"] = json_details["isEU"] if json_details else False
+    details["country_name"] = countries.get(details.get("country"))
+    details["isEU"] = details.get("country") in eu_countries if True else False
```

Comment:
I think you're doing the logic wrong here, it's doing `if True` which is always true.

Reply:
It's checking `if country is in eu_countries` return `True` else `False`

Reply:
```python
>>> 'asdf' if True else False
'asdf'
>>> 'asdf' if False else True
True
```

Reply:
```python
>>> eu_countries = ["PK","SE"]
>>> "PK" in eu_countries if True else False
True
>>> "SE" in eu_countries if True else False
True
>>> "ABC" in eu_countries if True else False
False
>>> 
```

Reply:
I think you missed the point - this is basically equivalent to always running the left-side expression which is `details.get("country") in eu_countries`.

Please read up on the python syntax for this expression.